src/chess/piece/Pawn.java

```diff
@@ -1,24 +1,81 @@
 package chess.piece;
 
-import java.util.HashMap;
-import java.util.Map;
+import chess.Position;
+
+import java.util.*;
 
 public class Pawn extends Piece {
   private static final Map<Color, Character> symbols = new HashMap<Color, Character>(){{
     put(Color.WHITE, '♙');
     put(Color.BLACK, '♟');
   }};
 
+  // if this pawn is moved, isFirstMoved turn out true but at next your own turn, it turn out false
+  // this variable is related to "en passant"
+  // ex. n:false -> n + 1:true -> n + 2:false
+  private boolean isFirstMoved;
```

Comment:
"en passant" only happens when the related Pawn had moved just before "en passant"
So how do you know the Pawn had moved just before or in the past?
I guess this parameter is not enough to handle "en passant"